### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jbostian**
Replace placeholder text with something useful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
-# python_ai_toolkit_zos
-Web interface for the Python AI Toolkit for z/OS
+# The Python AI Toolkit for IBM z/OS
+This is a library of Python packages that provide a foundation for much of the AI and 
+machine learning stack on IBM z/OS.  The information contained here can be used to manage 
+your Python deployment into development and production environments in your z/OS
+enterprise.
+
+Two other key resources are provided to effectively use this toolkit:
+- _**The package index**_.  For research and discovery of the functions available in the toolki.
```
Comment (zacburns) [inline README.md:8]: Typo on "toolki"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+  - [https://ibm-z-oss-oda.github.io/python_ai_toolkit_zos/](https://ibm-z-oss-oda.github.io/python_ai_toolkit_zos/)
+- _**The package repository**_.  Configure your pip environment to use this package repository
+rather than the default community pypi.org.  This can be accomplished in several ways.  See
+the Discussions for information on how to configure your environment appropriately.
+
+The _requirements_ directory of this repository contains a collection of 
+```requirements.txt``` files that represent the full contents of the toolkit at different 
+points in time.  They are released on a quarterly basis to provide a current package listing
+on a regular cadence.  Keep in mind that, like all open source projects, this toolkit is 
+dynamic, and evolves over time.  The requirements filesserve as a canonical list of the
```
Comment (zacburns) [inline README.md:18]: Typo "filesserve"

File: README.md (hunk 1 above)
Comment (jbostian) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (jbostian) [reply README.md]: Fixed.

**Review by zacburns — APPROVED** [review]

**Pull request merged**